syntax/index.html

```diff
@@ -414,7 +414,7 @@
       <ol>
         <li>metadata supplied by the user of the implementation that is processing the tabular data, see <a href="#overriding-metadata" class="sectionRef"></a>.</li>
         <li>metadata in a document linked to using a <code>Link</code> header associated with the <a>tabular data file</a>, see <a href="#link-header" class="sectionRef"></a>.</li>
-        <li>metadata located through a site-wide location configuration, see <a href="#site-wide-location-configuration" class="sectionRef"></a>.</li>
+        <li>metadata located through default paths which may be controlled by a site-wide location configuration, see <a href="#default-locations-and-site-wide-location-configuration" class="sectionRef"></a>.</li>
```

Comment:
That looks like a baby step in the right direction, but it still makes the mistake of not explicitly calling out the "standard metadata filename" feature on its own.  Also, the line above is not correct as written: default paths are *not* controlled by a site-wide location configuration, they may be *overridden* by a site-wide location configuration, just as *any* of the features in this list may be overridden by a feature with higher precedence.  

There is no benefit to the reader in attempting to portray the "standard metadata filename" as though it is a part of the "site-wide location configuration" feature, and there clearly is harm, because it makes the most commonly used feature hard to find.  From a reader's perspective, it is just plain misleading to portray it that way.  From the reader's perspective -- and from a technical perspective -- these features are completely separable: either of these features could have been included independently.  In fact, the "site-wide location configuration" feature was only added late in the standardization process, in response to guidance from the TAG.  It is *misleading* to phrase this as though these two features are inseparable.  The fact that both features are defined in the same section is fine; it is convenient to define them that way.  But please do not portray them as being inseparable, because that is not how users will see it, and it is not historically or technically accurate.  Those features are no more inseparable than any of the other features in the above list that take precedence over each other.